glfw/system_sdk.zig

```diff
@@ -216,6 +216,7 @@ fn determineSdkRoot(allocator: std.mem.Allocator, org: []const u8, name: []const
         return sdk_root_dir;
     } else |err| return switch (err) {
         error.FileNotFound => {
+            ensureGit(allocator);
```

Comment:
And this should be placed here: https://github.com/hexops/mach/blob/f0f91874da865d43eabb751905dd8e0f9026f09e/glfw/system_sdk.zig#L204 as both the code paths use git

Reply:
agreed